sympy/concrete/summations.py

```diff
@@ -253,6 +257,93 @@ def _eval_simplify(self, ratio, measure):
     def _eval_summation(self, f, x):
         return None
 
+    def is_convergent(self):
+        """
+        Convergence tests are used for checking the convergence of an
+        a series. There are various tests employed to check the
+        convergence. Like divergence test, root test, integral test,
+        alternating series test, comparison tests.
+
+        References
+        ==========
+
+        .. [1] https://en.wikipedia.org/wiki/Convergence_tests
+
+        Examples
+        ========
+
+        >>> from sympy import Interval, factorial, oo, S, Sum
+        >>> from sympy.abc import n
+        >>> Sum(n/(n - 1), (n, 4, 7)).is_convergent()
+        True
+        >>> Sum(n/(2*n + 1), (n, 1, oo)).is_convergent()
+        False
+        >>> Sum(factorial(n)/5**n, (n, 1, oo)).is_convergent()
+        False
+        >>> Sum(1/n**(S(6)/5), (n, 1, oo)).is_convergent()
+        True
+        """
+        from sympy import singularities, Interval
+        from sympy.solvers.solveset import solveset
+
+        _sequence_term = self.args[0]
+        sym = self.args[1][0]
+        _lower_limit = self.args[1][1]
+        _upper_limit = self.args[1][2]
+        _interval = Interval(_lower_limit, _upper_limit)
+
+        _sing = solveset(_sequence_term.as_numer_denom()[1], sym, S.Reals)
+        try:
+            for singul in _sing:
+                if singul.is_integer and singul in _interval:
+                    return S.false
```

Comment:
I think you should use regular True and False. S.true should only be used in places where a general symbolic boolean could be returned. Also, since this uses three-valued logic (with None), it should use `bool`, since S.true only represents two-valued logic.